frontend/apps/filemanager/filemanager.lua

```diff
@@ -1179,6 +1179,11 @@ function FileManager:onRefreshContent()
 end
 
 function FileManager:onShowFolderMenu()
+    local path_prefix = ""
+    if Device:isAndroid() then
+        local A, android = pcall(require, "android")  -- luacheck: ignore
+        path_prefix = android.getExternalStoragePath()
```

Comment:
That's `/storage/emulated/0` ?
A bit hard to figure what will be happening in the other part of the code below.
If we are in `/storage/emulated/0/koreader/Books/`, will we show in the button list only 3 items?:
```
/
(not) /storage
(not) /storage/emulated
(not) /storage/emulated/0
/koreader/
/koreader/Books/
```

Reply:
Three buttons with `(not)` are not shown.

Reply:
Feels a bit odd to hide some of them, and still show the real `/` ?
I understand all the `/storage/emulated/0/` are not really useful, but it helps understanding the tree - when hidden it's a bit odd. And some people may want to go into /storage/sdcard/ (if that exists, don't remember). And there's "Lock home folder" for people not wanting to see them.
I think they should show, with may be the ones unreadable in gray (so we can't go in them via the new button, as we can go directly into its parent or children).